frontend/src/handlers/editionHandler.ts

```diff
@@ -0,0 +1,17 @@
+import axios, { AxiosResponse } from "axios";
+import { AxiosConf } from "../api/calls/baseCalls";
+
+// Patch the name of an edition
+export async function editionSaveNameHandler(url: string, name: string): Promise<AxiosResponse> {
+    return await axios.patch(url, { name: name }, AxiosConf);
+}
+
+// Patch the year of an edition
+export async function editionSaveYearHandler(url: string, year: string): Promise<AxiosResponse> {
+    return await axios.patch(url, { year: year }, AxiosConf);
+}
+
+// Patch the active-state of an edition
+export async function editionSaveActiveHandler(url: string, active: boolean): Promise<AxiosResponse> {
+    return await axios.patch(url, { active: active }, AxiosConf);
+}
```

Comment:
Handlers should not call axios (since #400). Every axios call is in the api/calls module.

Reply:
Additionaly, please make use of the `base` functions (`basePath` for example)